docker-compose.yaml

```diff
@@ -29,7 +29,7 @@ services:
     volumes:
       - chainstorage:/data
 
-  kubo_ipfs:
+  ipfs:
```

Comment:
I don't think we have this everywhere, but some are the shorter one, so I did that here as well.

Reply:
Kubo is used by content publisher behind th scenes,
We might have to update env for IPFS_URL for content publisher